$OpenBSD: patch-src_Makefile,v 1.2 2002/01/20 19:54:13 naddy Exp $
# 1. Install on-line help to /usr/local/share/doc/icewm/doc
# 2. LD - was in original icewm port, I left it as it was
# 3. CXX - was in original icewm port, I left it as it was
--- src/Makefile.orig	Tue Oct  9 09:54:03 2001
+++ src/Makefile	Thu Jan 10 23:56:46 2002
@@ -20,7 +20,7 @@ CXXFLAGS = \
 	-DEXEEXT='"$(EXEEXT)"' \
 	-DICEWMEXE='"icewm"EXEEXT' \
 	-DICEHELPEXE='"icehelp"EXEEXT' \
-	-DICEHELPIDX='"$(DOCDIR)/icewm-$(VERSION)/icewm.html"' \
+	-DICEHELPIDX='"${DOCDIR}/icewm/doc/icewm.html"' \
         $(SYS_CFLAGS) $(SYS_INCDIRS)
 LFLAGS = $(SYS_LIBDIRS) 
 LIBS = $(SYS_LIBS)
@@ -124,8 +124,7 @@ clean:
 ################################################################################
 
 $(BINARIES):
-	@echo Linking $@...
-	@$(LD) -o $@ $($@_OBJS) $(LFLAGS) $($@_LFLAGS) $(LIBS) $($@_LIBS)
+	$(LD) -o $@ $($@_OBJS) $(LFLAGS) $($@_LFLAGS) $(LIBS) $($@_LIBS)
 
 libice.so: $(COREOBJS)
 	-@rm -f $@
@@ -142,11 +141,9 @@ wmabout.o: ../VERSION
 ################################################################################
 
 %.p: %.cc
-	@echo Analysing dependencies for $^...
-	@$(CXX) $(CXXFLAGS) -M -c $< > $@
+	$(CXX) $(CXXFLAGS) -M -c $< > $@
 
 %.o: %.cc
-	@echo Compiling $@...
-	@$(CXX) $(CXXFLAGS) -c $<
+	$(CXX) $(CXXFLAGS) -c $<
 	
 ################################################################################
